content/applications/productivity/studio/pdf_reports.rst

```diff
@@ -0,0 +1,340 @@
+===========
+PDF reports
+===========
+
+Odoo uses HTML and CSS technologies to create reports (PDF documents). HTML elements provide the
+basic structure, while CSS interacts with the elements to establish how they are shown to the user.
+Odoo also uses Bootstrap, a CSS framework, for the grid layout.
```

Comment:
not sure i'd mention that - people that understand that are usually the people who will *not* use studio to design reports, or you're going to give people the illusion that they can control more than what is actually possible. Perhaps simply specify that we use a grid system that can be controlled in the UI, but even then, it should pbly not be in the intro :)